src/main/java/bio/terra/cli/command/workspace/Update.java

```diff
@@ -11,7 +11,7 @@
 @Command(name = "update", description = "Update an existing workspace.")
 public class Update extends BaseCommand {
 
-  @CommandLine.ArgGroup(exclusive = false, multiplicity = "1..2")
+  @CommandLine.ArgGroup(exclusive = false, multiplicity = "1")
```

Comment:
My read of [the docs](https://picocli.info/apidocs/picocli/CommandLine.ArgGroup.html#multiplicity--), a value of `1` indicates "requires at least one", whereas `1..2` indicates that there can up to two instances of each. 

With the original `multiplicity`, usage message:
```
Error: Missing required argument(s): ([--name=<displayName>] [--description=<description>])
Usage: terra workspace update [--format=<format>] ([--name=<displayName>]
                              [--description=<description>])
                              [[--name=<displayName>]
                              [--description=<description>]]
Update an existing workspace.
      --description=<description>
                             workspace description
      --format=<format>      Set the format for printing command output: json,
                               text
                               Default: text
      --name=<displayName>   workspace display name
```

With this change:

```
Error: Missing required argument(s): ([--name=<displayName>] [--description=<description>])
Usage: terra workspace update [--format=<format>] ([--name=<displayName>]
                              [--description=<description>])
Update an existing workspace.
      --description=<description>
                             workspace description
      --format=<format>      Set the format for printing command output: json,
                               text
                               Default: text
      --name=<displayName>   workspace display name
```

Reply:
Done, here's the result:
```
Error: Missing required argument(s): ([--name=<displayName>] [--description=<description>])
Usage: terra workspace update [--format=<format>] ([--name=<displayName>]
                              [--description=<description>])
Update an existing workspace.
      --format=<format>      Set the format for printing command output: json,
                               text
                               Default: text
Property update parameters:
      --description=<description>
                             workspace description
      --name=<displayName>   workspace display name
```